docs/user_guide/HOWTO/Control_Repo.rst

```diff
@@ -27,242 +28,152 @@ control repository.  Before proceding with this section, you should
    You may find it helpful to read the section that explains how a control
    repository works in `Puppet, Inc.'s control repository documentation`_ .
 
-SIMP Omni-Environment Requirement
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Although (with a detailed understanding of SIMP internals) you can have a
-a :term:`SIMP server` that will function with just a Puppet environment, by
-default, a functioning SIMP server **requires** its entire SIMP Omni-Environment.
-So, the instructions in this HOWTO will create a SIMP Omni-Environment skeleton
-whose Puppet environment you can check into a branch in your control repository.
-
-The procedure in this HOWTO assumes the new SIMP Omni-Environment will be
-using its own secondary and writable environments.  For more complex SIMP
-Omni-Environment configurations (e.g., linking the secondary and writable
-directories to another environment in order to share rsync data and secrets),
-see the built-in SIMP CLI documentation for creating SIMP Omni-Environments:
-
-.. code-block:: bash
-
-   $ simp environment new -h
-
-Procedures
-^^^^^^^^^^
-
-STEP 1: Install Prerequisite Packages
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I think this section is important for those who install SIMP modules from internet repos, only, not from RPM or ISO.  Are we no longer describing that scenario?

Reply:
I think we need to add a second example under environments with this, I just didn't do it at this time.  I definately don't think it belongs in How to build a control repo.  I think we should add a send example that is more complicated that show how to set up a new env with a control repo that does not use the secondary environment of production.  I just don't know all the consequences of not using production and didn't want to half way do it and thought it could wait until 6.5

Reply:
I put in a ticket that has a 6.5 tag for this.  SIMP-6946